src/Datadog.Trace.ClrProfiler.Managed/Integrations/ElasticsearchNet5Integration.cs

```diff
@@ -129,7 +145,14 @@ public static class ElasticsearchNet5Integration
             }
             catch (Exception ex)
             {
-                Log.ErrorException($"Error resolving Elasticsearch.Net.IRequestPipeline.{nameof(CallElasticsearchAsync)}<T>(...)", ex);
+                Log.ErrorRetrievingMethod(
+                    exception: ex,
+                    moduleVersionPointer: moduleVersionPtr,
+                    mdToken: mdToken,
+                    opCode: opCode,
+                    instrumentedType: RequestPipelineInterface,
```

Comment:
nit: Perhaps here you could also rename the const string to `RequestPipelineInterfaceTypeName` so it's consistent with the other Elasticsearch integration.

Reply:
I would go further and use `RequestPipelineTypeName` (in both integrations). We don't need to distinguish this type as being an interface.